cmd/osmosisd/cmd/init.go

```diff
@@ -102,8 +102,6 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 			config.P2P.MaxNumOutboundPeers = 40
 			config.Consensus.TimeoutCommit = 2 * time.Second
 			config.Mempool.Size = 10000
-			config.StateSync.TrustPeriod = 112 * time.Hour
```

Comment:
Hrmm, this should be parameterized to be upperbounded by your unbonding period. Perhaps a bit lower, what is the default parameterizing it to?